Makefile

```diff
@@ -22,18 +22,81 @@ REGISTRY ?= gcr.io/heptio-images
 IMAGE = $(REGISTRY)/$(TARGET)
 DOCKER ?= docker
 DIR := ${CURDIR}
-VERSION ?= v0.1
+VERSION ?= v0.2
+
+ARCH    ?= amd64
+LINUX_ARCHS = amd64 arm64
+PLATFORMS = linux/amd64,linux/arm64
+
+IMAGEARCH ?=
+QEMUARCH  ?=
+
+MANIFEST_TOOL_VERSION := v1.0.0-rc
+
+ifneq ($(ARCH),amd64)
+IMAGE_TAG = $(IMAGE_REPO)-$(ARCH):$(IMAGE_TAG_NAME)
+endif
+
+ifeq ($(ARCH),amd64)
+IMAGEARCH =
+QEMUARCH  = x86_64
+else ifeq ($(ARCH),arm)
+IMAGEARCH = arm32v7/
+QEMUARCH  = arm
+else ifeq ($(ARCH),arm64)
+IMAGEARCH = arm64v8/
+QEMUARCH  = aarch64
+else ifeq ($(ARCH),ppc64le)
```

Comment:
You seem to support all these other args (OK) but I'm not sure how that plays with the default LINUX_ARCH and such. Could you clarify the intent?

If everything works for these architectures I'm inclined to just treat them all equally (build them all by default). My only concern is that we dont really run tests on a bunch of different architectures;  maybe @emuetzel can comment about the need to test each arch we're building.

Since it is so easy to cross-compile it seems impractical to have so many, varied resources to test.

Reply:
Hi @emuetzel
Any comments?